### DISCUSSION TITLE
How to download project

### DISCUSSION THREAD

**Discussion by tasnomani52** [dotnet/AspNetCore.Docs#26030] [General]
I amtrying to get the project
https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-mvc/intro/samples/cu-final

But dont see any link to download

**Comment by alierguc** [comment]
Hi my friend. You can extract the main file by downloading it. You can go to the file directory you downloaded and get that file. Of course, you will need to download the main file first.


[Main Project](https://github.com/dotnet/AspNetCore.Docs)
